locales/en.default.schema.json

```
@@ -146,6 +146,24 @@
         }
       }
     },
+    "search_input": {
+      "name": "Search input",
+      "settings": {
+        "header": {
+          "content": "Product suggestions"
+        },
+        "predictive_search_enabled": {
+          "label": "Enable product suggestions",
+          "info": "Affects the search bars in the header and on the search results page."
+        },
+        "predictive_search_show_vendor": {
+          "label": "Show vendor" 
```

```suggestion
          "label": "Show vendor",
          "info": "Visible when product suggestions enabled."
```

Based on @gretchen-willenberg suggestion!